eth/downloader/peer.go

```diff
@@ -115,13 +115,15 @@ func (w *lightPeerWrapper) RequestNodeData([]common.Hash) error {
 // newPeerConnection creates a new downloader peer.
 func newPeerConnection(id string, version int, peer Peer, logger log.Logger) *peerConnection {
 	return &peerConnection{
-		id:      id,
-		lacking: make(map[common.Hash]struct{}),
-
-		peer: peer,
-
-		version: version,
-		log:     logger,
+		id:                id,
+		lacking:           make(map[common.Hash]struct{}),
+		peer:              peer,
+		version:           version,
+		log:               logger,
+		headerThroughput:  float64(MaxHeaderFetch / 8),
+		blockThroughput:   float64(MaxBlockFetch / 8),
+		receiptThroughput: float64(MaxReceiptFetch / 8),
+		stateThroughput:   float64(MaxStateFetch / 8),
```

Comment:
Unless there's some measurable difference, I think starting from 0 is safer. (yeah, we might do a few more RTTs per peer per connection, but I don't think that's what we want to save on).